### DISCUSSION TITLE
documentation - explain, how to use the Pomodoro timer

### DISCUSSION THREAD

**Discussion by aisbergde** [ransome1/sleek#682] [Ideas]
from this link https://github.com/ransome1/sleek/wiki/Available-todo.txt-attributes-and-extensions#pomodoro-timer it is not clear, how to use the pomodoro timer.

> You choose a task, set a timer, and fully engage in it until the timer signals a break.

What does it mean? How to do? This is a very short instruction.

If I just set the pm:x then nothing happens. Only a pomodoro and the number x are shown. If I click on a task pomodoro, it looks like the task will be centered. But nothing else happens. No timer, no status information, nothing.

Please extend the documentation and explain in more detail, how to use the pomodoro feature.

**Comment by ransome1** [comment]
@aisbergde there is no real functionality connected to the pomodoro timer. It has simply been added to establish compatibility. Some users are incorporating the pomodoro system into their todo.txt system and needed it to be accepted by sleek to use it for sorting and visualisation.

**Comment by aisbergde** [comment]
Thank you. I thought, I missed something, for example how to start the timer :-)
